src/pybind/mgr/nfs/export.py

```diff
@@ -637,6 +640,12 @@ def create_cephfs_export(self,
                              access_type: str,
                              clients: list = [],
                              sectype: Optional[List[str]] = None) -> Tuple[int, str, str]:
+        fs = fs_client.get_fs(fs_name=fs_name)
+        stx = fs.statx(path, cephfs.CEPH_STATX_MODE,
+                       cephfs.AT_SYMLINK_NOFOLLOW)
```

Comment:
It seems the `fs.statx()` will raise as not found, should you put this in the `try` scope ?

Reply:
Yeah this needs a overhaul, the above code doesn't work, i'm working on fixing this.

Reply:
wrapped the code into try-except block, PTAL